specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -0,0 +1,151 @@
+
+# VEP-994: Jupyter Notebook Integration
+
+* **Author(s):** Duygu Hasan(hduyg@vmware.com)
+* **Status:** draft
+
+- [Summary](#summary)
+- [Glossary](#glossary)
+- [Motivation](#motivation)
+- [Requirements and goals](#requirements-and-goals)
+- [High-level design](#high-level-design)
+- [API Design](#api-design)
+- [Detailed design](#detailed-design)
+- [Implementation stories](#implementation-stories)
+- [Alternatives](#alternatives)
+
+## Summary
+
+<!--
+Short summary of the proposal. It will be used as user-focused
+documentation such as release notes or a (customer facing) development roadmap.
+The tone and content of the `Summary` section should be
+useful for a wide audience.
+-->
+
+## Glossary
+<!--
+Optional section which defines terms and abbreviations used in the rest of the document.
+-->
+
+## Motivation
+
+Versatile Data Kit provides utilities for developing SQL and Python data jobs and deploying them in the cloud. However, the only interface it has for development is the CLI.
+
+As a workload users install their own IDE (PyCharm, for example) locally and configure it with VDK. However, VDK is used by people who do not want to nor need to learn how to work with CLI and install and configure IDEs.
+
+Moreover, many of our users are data scientists who work with big data, and they work with notebooks to visualize and test the data. In order to do that, they copy their data job code from the IDE and paste it to a notebook to see the visualization or test it then copy the changes on the notebook and paste it to the IDE.
+
+It would be much easier for the users to just open a familiar UI and enter their SQL queries or Python code, without using the CLI and without copy pasting code from one place to another.
```

Comment:
I think  you can explain in more details some of the challenges we have found that we could address 
 - rerun whole job for small changes or re-run failing steps, testing small sections,